app/pages/Project/index.js

```diff
@@ -112,13 +116,35 @@ class ProjectPage extends Component {
 
     const {
       preview,
+      previewSize,
       addAlert,
     } = this.props
 
     await fsWriteFile(p, preview.content)
     addAlert('Successfully exported HTML', 'success')
     this._filelist.refresh()
   }
+  
+  handleScreenshot = async () => {
+    const {
+      preview,
+      previewSize,
+      addAlert,
+      location,
+    } = this.props
+    
+    const filename = pathModule.basename(this.state.activeFile.name, `.mjml`) 
+  
+    const [mobileWidth, desktopWidth] = [previewSize.get('mobile'), previewSize.get('desktop')]
+
+    const [mobileScreenshot, desktopScreenshot] = await Promise.all([takeScreenshot(preview.content, mobileWidth), takeScreenshot(preview.content, desktopWidth)])
+
+    await fsWriteFile(`${location.query.path}/${filename}-mobile.png`, mobileScreenshot)
+    await fsWriteFile(`${location.query.path}/${filename}-desktop.png`, desktopScreenshot)
+    
```

Comment:
like above
```js
await Promise.all([
  fsWriteFile(...),
  fsWriteFile(...),
])
```

and also, files paths needs to be joined with `path.join` because our friend Windows use `\`

```js
path.join(location.query.path, `${filename}-mobile.png`)
```